geochem_inverse_optimize.py

```diff
@@ -245,6 +251,29 @@ def get_unique_upstream_areas(sample_network: nx.DiGraph):
     return areas
 
 
+def calibrate_regularizer(
```

Comment:
This function doesn't actually calibrate anything, it just makes a plot. Maybe change the name to reflect that, eg `plot_sweep_of_regularizer_strength`.

Reply:
Agreed. Changed as suggested.